packages/integration-sdk-testing/src/executeStepWithDependencies.ts

```diff
@@ -9,16 +9,17 @@ import {
 export async function executeStepWithDependencies(params: StepTestConfig) {
   const { stepId, invocationConfig, instanceConfig } = params;
 
-  if (invocationConfig.dependencyGraphOrder) {
```

Comment:
The Google Cloud integration has a limited number of steps that are set with `dependencyGraphId: 'last'`, but the remaining steps do _not_ have a `dependencyGraphId` and inherently execute first.

While we cannot presently use the `executeStepWithDependencies` function on any of the steps with `dependencyGraphId: 'last'`, there's no reason we should be excluded from using it with any of the other steps in that integration.